aws/templates/id/id_lb.ftl

```diff
@@ -85,6 +85,52 @@
                         "Default" : 604800
                     }
                 ]
+            },
+            {
+                "Name" : "Redirect",
```

Comment:
I am using presence of Redirect or Fixed conf sections to indicate they are required, which effectively acts as the action property. If neither is present, then forward is the fault.

What advantage would adding an extra flag give us?

Reply:
I could think of a couple of reasons
 - Setting a preference if both have been set
 - Could be an easy way to do the redirect vs forward swapping. You configure the forward and the redirect and then pick the one to use at that time

Reply:
That means redeploying the rule rather than simply changing its priority which is what I am working with now. Re both set, fixed will override redirect - but if both are there, then its a configuration error.

Reply:
Okey doke sounds good, just wanted to make sure had the configuration error covered